dbt/tests/generic/test_column_length.sql

```diff
@@ -8,14 +8,16 @@
 -- additional_select_columns argument.
 {% test column_length(model, columns, length, additional_select_columns=[]) %}
 
-    {%- set columns_csv = additional_select_columns | join(", ") %}
+    {%- set additional_select_columns_csv = format_additional_select_columns(
+        additional_select_columns
+    ) %}
 
     {%- set length_columns = [] %}
     {%- for column in columns %}
         {%- set length_columns = length_columns.append([column, "len_" + column]) %}
     {%- endfor %}
 
-    {%- set select_columns = additional_select_columns %}
+    {%- set select_columns = [] %}
```

Comment:
The previous version of this code was somewhat convoluted; we set `columns_csv` to a parsed version of `additional_select_columns` on line 11, but then redefined `columns_csv` on line 36 to a list of transformed strings including (among others) columns from `additional_select_columns`. I've gone ahead and simplified this logic so that we only parse `additional_select_columns` once, and then template it directly in the `SELECT` query alongside `columns_csv` on lines 41-49 below.